infra/charts/feast-feature-server/templates/instrumentation.yml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.metrics }}
+apiVersion: opentelemetry.io/v1alpha1
+kind: Instrumentation
+metadata:
+  name: feast-instrumentation
+  namespace: {{ .Release.Namespace }}
```

Comment:
this line shouldn't be necessary ... helm should deploy to the appropriate namespace. the existing templates don't specify this line, so we should probably follow suit